src/game/moveFinding/findKnightBaseMoves.js

```diff
@@ -0,0 +1,27 @@
+import BoardCoordinates from "../board/boardCoordinates";
+class Knight {
+    
+
+
+    findMovesx(boardSituation, startingCoordinates, pieceColor){

```

Comment:
Zasadniczo, to chodziło nam tu o funkcję, nie klasę (tzn. w JS klasa to specyficzna funkcja, ale roozumiesz, o co mi chodzi) - i dobrze jakby nazwa tej funkcji była jak "zadana" (i pasująca do nazwy pliku).

Plus, oczywiście, trzeba ją wyeksportować.